swift/Tests/Blockchains/OntologyTests.swift

```diff
@@ -53,12 +53,12 @@ class OntologyTests: XCTestCase {
         let input = TWOntologySigningInput.with {
             $0.contract = "ONT"
             $0.method = "transfer"
-            $0.ownerPrivateKey = Data(hexString: "4646464646464646464646464646464646464646464646464646464646464646")!
```

Comment:
I don't think `Data(hexString:)` is same as `String.data`

Reply:
Right, we pass swift hexString to cpp, then cpp will convert it to bytes to construct a private key.

```
$0.ownerPrivateKey = "4646464646464646464646464646464646464646464646464646464646464646".data(using: .utf8)!
```

Reply:
why not just pass bytes? it's not consistent with other implementations

Reply:
okay, we will change it.